config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -0,0 +1,44 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: gcp-k8s-cluster-bundle-presubmit
+    always_run: true         # Run for every PR, or only when requested.
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180918-762ed5b64-master
+        imagePullPolicy: Always
+        command:
+        - "bazel"
+        - "test"
+        - "..."
+postsubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: gcp-k8s-cluster-bundle-postsubmit
```

Comment:
Similar to above: `post-k8s-cluster-bundle-bazel-test`.

Reply:
Okay